Documentation/README_ManipulationHandler.md

```diff
@@ -78,6 +78,15 @@ Specifies on which axis the object will rotate when interacted with.
 **Constraints on Movement**
 * *None*
 * *Fix distance from head*
+* *Translation Axes* - If this option is selected, movement will be constrained to the axes specified in "Translation Axes"
```

Comment:
It would be good to specify here, and in comments, if this axis is in the object's space "local space", or the world space. It looks like the constraint is in world space, since if the constraint is set to just be X, then the object will have only its x value changing, which is a world space value.